docs/psmysql-pro.md

```diff
@@ -0,0 +1,22 @@
+# Percona Server for MySQL Pro
+
+Percona Server for MySQL Pro includes [solutions](#solutions) typically demanded by large enterprises. Percona Server for MySQL Pro is wrapped in packages created and tested by Percona. These packages are supported only for Percona Customers with a subscription.
```

Comment:
MySQL Pro is wrapped in packages?
Maybe
MySQL Pro is contains packages created and tested by Percona.